### ISSUE TITLE
Facebook API version 2.4 is not supported

### ISSUE THREAD

**Issue by vladas21** [chute/Chute-SDK-V2-Android#10]
Facebook published the latest API v.2.4 to every new application, but this API is not supported by current version of Chute PhotoPickerPlus (both Android and iOS version). 

The problem is critical as none of new apps can introduce Facebook authorization in its functionality until this issue is resolved.

**Comment by darko1002001** [comment]
Can you share some insights to where you read this? I am unable to find any info on this.

Our mobile apps and libraries don't use the facebook sdk's directly, the server contacts the facebook API directly to make the requests. As far as i know their previous version is still functional but deprecated for at least 2 more years. And when we change the API's to communicate with the newest version facebook released any mobile apps using the sdk and pp+ won't have to change anything.

**Comment by vladas21** [comment]
Probably, I submitted issue on wrong location.. Sorry if I'm wrong.

I will try to explain problem in details.

I've created the new Facebook application and by default application use the latest published API (currently v2.4). There is no possibility do change it (like in this case to downgrade it from v2.4 to v2.3).

I've integrated ChutePhotoPickerPlus to my Android and iOS application, and also configured to use my CLIENT_ID/CLIENT_SECRET.

The login dialog shows regularly (user is able to specify his email and password). When user clicks on login button the the following error appears:

Invalid Scopes: offline_access. This message is only shown to developers. Users of your app will ignore these permissions if present. Please read the documentation for valid permissions at: https://developers.facebook.com/docs/facebook-login/permissions

I did a lot of debugging of my application, tried almost every combination of setting on my Facebook development account but nothing gives positive result. 

When I try with CLIENT_ID/CLIENT_SECRET which is given in sample project of PhotoPickerPlus:

APP_ID = "4f3c39ff38ecef0c89000003";
APP_SECRET = "c9a8cb57c52f49384ab6117c4f6483a1a5c5a14c4a50d4cef276a9a13286efc9";

It works as expected.

Finally during the debugging process, I've started to monitor URLs which are invoked during the login process and wrote them into a log file - both for my Facebook client and for client from examples. Only difference was in path segment: /v2.4/ - for my version and /v2.3/ for the working version. PhotoPickerPlus, uses Chute SDK, and this component communicate with server. I assume that server generates URL for client, but I'm not sure whether Chute SDK plays important role for this links. That is the reason I've created this issue right here.

So, If I understand correctly, maybe only the getchute.com server should be updated (because the server uses Facebook SDK directly)?

**Comment by darko1002001** [comment]
Yes it seems the server will need to be updated if this is causing the issues for new facebook apps OAuth login.

You can also try not setting the facebook app id and app secret for Chute in the admin and just using the ones we provide if that way the login works for your Chute APP id and App secret/
